### PR TITLE
(MH] 🐛🌐  AU update incorrect Plurals/Namrs

### PR DESCRIPTION
**Pull request by Gooochy**
Btw there is 2 SES units and 2 TOG Units on here so that needs fixing but im not really sure how

### CODE REVIEW

File: src/modules/missionHelper/i18n/en_AU.json [hunk 1]
```diff
@@ -45,27 +45,27 @@
...
-			"platform_trucks": "Turntable Ladder trucks",
+			"platform_trucks": "Turntable Ladder Truck | Turntable Ladder Trucks",
 			"police_cars": "Police Car | Police Cars",
 			"police_helicopters": "Police Helicopter | Police Helicopters",
 			"police_horse": "Mounted Police",
 			"swat_armored_vehicle": "TOG Armoured Vehicle | TOG Armoured Vehicles",
 			"swat_suv": "TOG SUV | TOG SUVs",
 			"water_rescue": "SES Vehicle | SES Vehicles",
-			"water_tankers": "Water Carrier | Water Carriers"
+			"water_tankers": "Bulk Water Tanker | Bulk Water Tankers s"
```
Comment (JRH-1997) [inline src/modules/missionHelper/i18n/en_AU.json:68]: That `s` on the end is not correct I think?

File: src/modules/missionHelper/i18n/en_AU.json [hunk 2]
```diff
@@ -45,27 +45,27 @@
...
 			"brush_truck": "Brush Truck | Brush Trucks",
-			"fire_aviation": "Airborne Firefighting | Airborne Firefightings",
+			"fire_aviation": "Airborne Firefighting Vehicle | Airborne Firefighting Vehicles",
 			"firetrucks": "Fire Engine | Fire Engines",
-			"gw_wasserrettung": "SES Unit",
-			"hazmat_vehicles": "HAZMAT | HAZMATS",
-			"heavy_rescue_vehicles": "Rescue",
-			"helicopter": "Air Ambulance",
+			"gw_wasserrettung": "SES Unit | SES Units",
+			"hazmat_vehicles": "HAZMAT Truck | HAZMATS Trucks",
```
Comment (JRH-1997) [inline src/modules/missionHelper/i18n/en_AU.json:54]: Should this not be `HAZMAT` without `s` in plural? So `HAZMAT Trucks`

Comment (Gooochy) [reply src/modules/missionHelper/i18n/en_AU.json]: Corrected but can you take a look how there is 2 SES and 2 TOG

File: src/modules/missionHelper/i18n/en_AU.json [hunk 3]
```diff
@@ -45,27 +45,27 @@
...
 			"boats": "Boat | Boats",
 			"brush_truck": "Brush Truck | Brush Trucks",
-			"fire_aviation": "Airborne Firefighting | Airborne Firefightings",
+			"fire_aviation": "Airborne Firefighting Vehicle | Airborne Firefighting Vehicles",
 			"firetrucks": "Fire Engine | Fire Engines",
-			"gw_wasserrettung": "SES Unit",
-			"hazmat_vehicles": "HAZMAT | HAZMATS",
-			"heavy_rescue_vehicles": "Rescue",
-			"helicopter": "Air Ambulance",
+			"gw_wasserrettung": "SES Unit | SES Units",
```
Comment (JRH-1997) [inline src/modules/missionHelper/i18n/en_AU.json:53]: This one is not existing in en_AU

Comment (Gooochy) [reply src/modules/missionHelper/i18n/en_AU.json:53]: Should I delete it?

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**